test/models/name_test.rb

```diff
@@ -3727,6 +3727,35 @@ def test_more_brief_authors
     assert_equal("(A et al.) D et al.", name.send(:brief_author))
   end
 
+  # ----------------------------------------------------
+  #  Name:Observations
+
+  def test_where_proposed
```

Comment:
Is there a better file, or a better location in this file, for this test?

Reply:
There may be a Name::ObservationsTest?

Reply:
Nope. But the module tests could be broken out like that. NameTest seems ripe for a refactor. In another PR.

Reply:
Oh there is already a name_parse test and a name_sorter test. This could be in its own file, I'd lean that way.

Reply:
Thanks for looking and for the advice. I moved it to its own file.